tests/unit/api/test_targets.py

```diff
@@ -324,3 +324,48 @@ def test_post_forbidden_user_incorrect_scope_token(self, test_client):
         assert response.json() == {
             "detail": {"error": "scope 'delete:targets' not allowed"}
         }
+
+
+class TestPostTargetsPublish:
+    def test_post_publish(self, monkeypatch, test_client, token_headers):
+        url = "/api/v1/targets/publish/"
+
+        mocked_repository_metadata = pretend.stub(
+            apply_async=pretend.call_recorder(lambda *a, **kw: None)
```

Comment:
Wondering when you mock when do you need both `*a` and `**kw` in the lambda function?
Right now I see that for `apply_async` you use keyword arguments only. 
Why do you also need `*a`?